### PR TITLE
Added IPv4/IPv6 filter support for tcp trace tools

### PR DESCRIPTION
**Pull request by hariharan-a**
Added command line optional argument [-4 | -6] similar to ping/traceroute to support filtering IPv4/IPv6 tcp stats. 

+ @brendangregg

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: tools/tcpconnlat.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ def positive_float(val):
     ./tcpconnlat -t        # include timestamps
     ./tcpconnlat -p 181    # only trace PID 181
     ./tcpconnlat -L        # include LPORT while printing outputs
+    ./tcpconnlat -4        # trace IPv4 family only
+    ./tcpconnlat -6        # trace IPv6 family ony
```
Comment (yonghong-song) [inline tools/tcpconnlat.py:44]: family only

**Review by yonghong-song — COMMENTED** [review]
I think adding -4 -6 make sense as a lot of tools already have "-4" "-6" options like ping, ss etc. Could you also add new options to man pages? Thanks!

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: tools/tcpconnlat.py (hunk 1 above)
Comment (hariharan-a) [reply tools/tcpconnlat.py]: Done. Fixed typo and added options to man pages.

**Comment by brendangregg** [comment]
Looks good, thanks, and follows the precedent of -4/-6 from other network tools. Thanks for updating all the man pages and example files.

**Comment by anakryiko** [comment]
@hariharan-a Would you be able to check libbpf-tools and see about adding the same feature there?

**Comment by hariharan-a** [comment]
> @hariharan-a Would you be able to check libbpf-tools and see about adding the same feature there?

Sure. I will check.

**Pull request merged**